### PR TITLE
Project-weather-app

### PR DESCRIPTION
**Pull request by Looziolooz**
## Netlify link
Add your Netlify link here.
PS. Don't forget to add it in your readme as well.
https://lorenzoweatherapp.netlify.app/

## Collaborators
Add your collaborators here. Write their GitHub usernames in square brackets. If there's more than one, separate them with a comma, like this:
[github-username-1, github-username-2]

### CODE REVIEW

File: fetch2.js [hunk 1]
```diff
@@ -0,0 +1,169 @@
+const apiKey = '3575626ed7c8ddf8baca764025081251';
+const baseUrl1 = 'https://api.openweathermap.org/data/2.5';
+const baseUrl2 = 'https://api.openweathermap.org/data/2.5';
+
+function fetchWeather(city) {
```
Comment (AntonellaMorittu) [inline fetch2.js:5]: although this is correct, please note during the bootcamp we're using the Arrow Functions (const () => {}) approach to define functions 👍

**Review by AntonellaMorittu — CHANGES_REQUESTED** [review]
Ciao Lorenzo, very good job this week! Your code is well structured and the app is working as it should - almost. Have you tested the search functionality with different cities? Sunrise and sunset times need to be adjusted :) That was a stretch goal so you don't need to fix it to pass the assignment.

However, please pay attention to these basic requirements:

- The app should have: 4-day forecast You are displaying all 5 days when we requested 4 
- the temperature should be rounded to 1 decimal 
- the app should follow as close as possible the chosen design

Fix these things and you're good to go:) Keep up the awesome work 🚀

File: script.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      document.querySelector('.description').textContent = data.weather[0].description;
+
+      // Update weather icon
+      const iconCode = data.weather[0].icon;
+      const iconUrl = `http://openweathermap.org/img/wn/${iconCode}.png`;
+      document.querySelector('.Weather-icon').src = iconUrl;
+
+      // Convert sunrise and sunset times from milliseconds to a readable format
+      const sunriseTime = new Date(data.sys.sunrise * 1000).toLocaleTimeString([], { hour: '2-digit', minute: '2-digit' });
+      const sunsetTime = new Date(data.sys.sunset * 1000).toLocaleTimeString([], { hour: '2-digit', minute: '2-digit' });
```
Comment (Maria-Manuela) [inline script.js]: Sunrise and sunset it is not displayed, check the element IDs  in the HTML if there are accessible to JS code

File: script.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+document.querySelector('.search button').addEventListener('click', function() {
+  const input = document.querySelector('.search input').value;
+  fetchWeather(input);
+});
+
+// You can also listen for the 'Enter' key press event to trigger the search
+document.querySelector('.search input').addEventListener('keypress', function(event) {
+  if (event.key === 'Enter') {
```
Comment (Maria-Manuela) [inline script.js]: the code snippet is well-written and serves its purpose effectively

**Review by Maria-Manuela — COMMENTED** [review]
Well done, Lorenzo! The code is well-written and functional. However, there are a few areas that need attention. Address the errors appearing in the console, fix the display of sunrise and sunset times, and ensure that you follow the project instructions.

File: fetch2.js (hunk 1 above)
Comment (Looziolooz) [reply fetch2.js]: Hi Antonella, thank you for pointing that out. Updated✔️

File: script.js (hunk 2 above)
Comment (Looziolooz) [reply script.js]: Hi Maria, I had left them hidden so I could edit them later. They are now visible. Thank you!

File: script.js (hunk 3 above)
Comment (Looziolooz) [reply script.js]: Thank you!!

**Review by HIPPIEKICK — APPROVED** [review]

**Pull request closed without merging**